user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -4,19 +4,28 @@ uid: Register_Catalog_Item
 
 # Registering a Catalog item
 
-Below you can find detailed information on how you can register a private Catalog item, either [using the Catalog UI](#using-the-catalog-ui), or [using the Catalog API](#using-the-catalog-api).
+In order to make a new Catalog item or a new version of a Catalog item available in the [DataMiner Catalog](https://catalog.dataminer.services/), you need to register it with an organization.
+
+When you register a new Catalog item, it will only become available for the registered organization, and it will be marked as *private* in the UI to indicate this.
```

Comment:
```suggestion
When you register a new Catalog item, it will only become available for the registered organization, and it will be marked as *private* on the Catalog to indicate this.
```